### PR TITLE
Added diagram for tool, minor cleanup, "exercise" at end

### PR DESCRIPTION
**Pull request by Purg**
Added a simple sequence diagram  for the utility and added a few more words to some explanations.

In order to accommodate CI execution, the parameterization needed to be lowered to fit below 7GB of RAM usage [available to free CI runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources). An "additional example" as previously demoed as been adapted to be an "exercise for the user".

There is no additional change note due to modifying something that is already represented.

### CODE REVIEW

**Review by brianhhu — COMMENTED** [review]
Some minor typos in the last cell:

The above saliency heatmap is clearly very **coarse**.

Try executing the following cell, which **instantiates** an alternate RISEGrid instance

Was there a reason why we needed to bump up the minimum version of torch/torchvision? I don't think it will cause issues, but most of our other notebooks are pinned to 1.9.0/0.10.0.

**Comment by Purg** [comment]
> Was there a reason why we needed to bump up the minimum version of torch/torchvision? I don't think it will cause issues, but most of our other notebooks are pinned to 1.9.0/0.10.0.

The reason I touched that was to "deconflict" the xaitk-saliency lockfile version from the requirement in `smqtk-detection[centernet]` extra which is currently set to require `torch=^1.10.0` and `torchvision=^0.11.1`. Given your comment and re-applying my brain-cells, I realized that I could, for this notebook, just install `smqtk-detection[centernet]` with the `-U` option. I will revert the pyproject update for another time if appropriate.

**Review by schencej — APPROVED** [review]
Other than the one minor comment, LGTM.

**Comment by brianhhu** [comment]
Pending the minor fix that Elim mentioned above, it LGTM- thanks!

File: examples/SerializedDetectionSaliency.ipynb [hunk 1]
```diff
@@ -83,7 +107,9 @@
     "from xaitk_saliency.impls.perturb_image.rise import RISEGrid\n",
     "from xaitk_saliency.impls.gen_detector_prop_sal.drise_scoring import DRISEScoring\n",
     "\n",
-    "image_perturber = RISEGrid(n=50, s=8, p1=0.5)\n",
+    "# Coarse Parameterization to allow for a abbreviated live demonstration\n",
+    "# - Will generate 20 perturbed images with a coarse 8x8 grid masked regions.\n",
+    "image_perturber = RISEGrid(n=20, s=8, p1=0.5, seed=0)\n",
```
[the remark this answers is not in the document]
Comment (brianhhu) [reply examples/SerializedDetectionSaliency.ipynb]: @Purg Still need to address this minor change above, changing `n` from 50 -> 20.

If you're already in the notebook editing, I also noticed two really minor changes:

Second sentence: "The following figure highlights the inputs and outputs of this utility **function**."

Last sentence: "The following **requires** more memory resources **than** the previous parameterization to accommodate the larger matrix multiplication operations performed."

[the remark this answers is not in the document]
Comment (Purg) [reply examples/SerializedDetectionSaliency.ipynb]: Apologies for the delay. Fix update incoming.

**Review by brianhhu — APPROVED** [review]
LGTM, thanks!

**Comment by Purg** [comment]
Autosquashed. Will merge after appropriately green CI.

**Pull request merged**